### ISSUE TITLE
Setup ravencoin ?

### ISSUE THREAD

**Issue by dingodingo2** [UnamSanctam/UnamWebPanel#70]
Hey ! I would like to know if there is any way to setup a "default ravencoin" config on the panel. And also, is nanopool good for this coin ? Thanks :)

**Comment by UnamSanctam** [comment]
Ravencoin is XMRig so "Default xmrig" would be used for it. Though ETH or ETC are more profitable and less detectable than RVN so either ETH or ETC (which requires less VRAM than RVN as well) is usually recommended instead.

**Comment by dingodingo2** [comment]
Yeah but the issue is that in June eth will go p.o.s

**Comment by UnamSanctam** [comment]
Yes but not ETC, currently ETC is both more profitable, takes less VRAM and is more stealthily (due to the algorithm) than RVN.

**Comment by dingodingo2** [comment]
Oh that's great! and for etc nanopool I guess ?

**Comment by UnamSanctam** [comment]
Yes or ethermine (which most use).

**Comment by dingodingo2** [comment]
So I've got an issue. Similar to nanopool before but, I put the json and nothing update. I tried adding the algo field, the password field but it's not working. If you have any idea what could cause the issue... 

`{"pool":"eu1-etc.ethermine.org","port":4444,"wallet":"wallethere"}`

**Comment by UnamSanctam** [comment]
That's the XMR miner format and not ETH miner format, you can find the ETH format here: https://github.com/UnamSanctam/SilentCryptoMiner/wiki/ETH-Miner#remote-configuration or you can use the builder to automatically create the JSON by entering the settings for your miner in the builder and then going to the JSON tab.

**Comment by UnamSanctam** [comment]
Yes it's the worker name, you can use `{COMPUTERNAME}` to use the computers name as the worker name.

**Comment by dingodingo2** [comment]
So I used the random tag but they all have the same username. That's strange...

`{"url": "stratum://wallet.{RANDOM}@eu1-etc.ethermine.org:4444","algo": "etchash"}`

and in the optional settings of the miner I did put {RANDOM} in worker name. But they all have the same worker name... (for example : frADBc3Qsa)

**Comment by UnamSanctam** [comment]
I would recommend {COMPUTERNAME} instead since otherwise they'll randomize to a new value on each startup (or connection change).
